svgeditor/scripts.js

```diff
@@ -71,14 +71,25 @@ const updateDrawing = {
     const text = svg.getElementById(textId);
     // this replaces the existing text with whatever is passed into to the updateDrawing.updateSvg function
     text.textContent = newText;
+    // and then updates the canvas to match
+    updateDrawing.updateCanvas(svg.querySelector('svg'));
+
   },
-  updateCanvas: function() {
+  updateCanvas: function(svg) {
+    svg = makeBlobFromSVG(svg);
+    let c0 = document.getElementById(canvasId);
+    let ctx0 = c0.getContext("2d");
+    const image = new Image();
+    image.addEventListener('load', () => {
+      ctx0.drawImage(image, 0, 0, c0.width, c0.height);
+    }, false);
```

Comment:
I noticed that this is the only event listener that deals with event bubbling. I don't fully understand that concept but it seems interesting that here you throw the false in but it's not needed elsewhere. Is this make or break? Like without it would it still work?